project.clj

```diff
@@ -57,7 +57,7 @@
                  ;; Database connectivity
                  [com.zaxxer/HikariCP "2.4.3"]
                  [org.clojure/java.jdbc "0.4.2"]
-                 [org.postgresql/postgresql "9.4-1206-jdbc4"]
+                 [org.postgresql/postgresql "9.4.1208"]
```

Comment:
Housekeeping, there was a ticket right next to this one for updating to 9.4, and I noticed we weren't using the jdbc driver versions that are on the postgres page